roadmap.md

```diff
@@ -0,0 +1,35 @@
+Roadmap Zombienet v2
+
+## Infra
+- Chaos testing, add examples and explore possibilities in `native` and `podman` provider
+- Add `docker` provider
+- Add `nomad` provider
```

Comment:
:+1:

Reply:
Hi @drahnr, did you have time to make a quick chat to review some request relates to nomad?
Thanks!